protostar/starknet/cheatable_state.py

```diff
@@ -87,6 +87,7 @@ async def execute_entry_point_raw(
             entry_point_type=EntryPointType.EXTERNAL,
             calldata=calldata,
             caller_address=caller_address,
+            initial_gas=10**10,
```

Comment:
Repeated

Reply:
[Use constant from cairo-lang instead of coming up with a number](https://github.com/software-mansion/protostar/pull/1526/commits/250c3bd0cd85b2ab0f41c7541aa46e7d91fe7bf4)